courses/ahybrid/v1.0/AHYBRID050/scripts/create_gke.sh

```diff
@@ -16,69 +16,65 @@
 
 source ./scripts/env.sh
 
-# create the gke cluster
 gcloud config set compute/zone ${C1_ZONE}
 gcloud beta container clusters create ${C1_NAME} \
     --machine-type=n1-standard-4 \
     --num-nodes=4 \
     --workload-pool=${WORKLOAD_POOL} \
     --enable-stackdriver-kubernetes \
     --subnetwork=default \
-    --labels=mesh_id=${MESH_ID} \
-    --release-channel=regular
+    --labels mesh_id=${MESH_ID}
 
 # service account requires additional role bindings
 kubectl create clusterrolebinding [BINDING_NAME] \
     --clusterrole cluster-admin --user [USER]
 
-# create the service account for gke-connect
 gcloud iam service-accounts create connect-sa
 
-# assign GSA the role it needs
 gcloud projects add-iam-policy-binding ${PROJECT_ID} \
  --member="serviceAccount:connect-sa@${PROJECT_ID}.iam.gserviceaccount.com" \
  --role="roles/gkehub.connect"
 
-# download the service account key
 gcloud iam service-accounts keys create connect-sa-key.json \
   --iam-account=connect-sa@${PROJECT_ID}.iam.gserviceaccount.com
 
-# register the cluster
 gcloud container hub memberships register ${C1_NAME}-connect \
    --gke-cluster=${C1_ZONE}/${C1_NAME}  \
    --service-account-key-file=./connect-sa-key.json
 
-# config project for Anthos Service Mesh
```

Comment:
I personally think this is the one step where it isn't clear what's happening; I sort of prefer the comment to remain

Reply:
I can add this back in

Reply:
added